manta-benchmark/benches/to_private.rs

```diff
@@ -14,26 +14,30 @@
 // You should have received a copy of the GNU General Public License
 // along with manta-rs.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Mint Benchmarks
+//! To Private Benchmarks
```

Comment:
Keep this file and `private_transfer.rs` "the same shape", like don't change the variable name conventions. In here you use `transferpost` instead of `to_private`, etc.